### DISCUSSION TITLE
How to remove cloud sync data from Google Drive

### DISCUSSION THREAD

**Discussion by JD99** [sienori/Tab-Session-Manager#552] [Q&A] [answered]
I apologize in advance for bad English. 
I start synchronization. 
Half the synchronization freezes. 
restarts only after browser reboot.
no files in google drive
firefox 76.0.1

![Скриншот 04-06-2020 230905](https://user-images.githubusercontent.com/8160891/83805395-7bd56f80-a6b8-11ea-9b7a-fdc80e31267c.png)
![Скриншот 04-06-2020 230838](https://user-images.githubusercontent.com/8160891/83805397-7c6e0600-a6b8-11ea-94b8-2f014cf32357.png)

**Comment by sienori** [comment, marked as answer]
If the error persists, try deleting the data.
Open your Google Drive settings and delete the TSM data.

![image](https://user-images.githubusercontent.com/29514424/85859580-5986cb80-b7f8-11ea-958c-022dd9f3eccf.png)
![image](https://user-images.githubusercontent.com/29514424/85859478-39efa300-b7f8-11ea-9082-017461d1197e.png)

**Comment by JD99** [comment]
thank. works

**Comment by makarichevss** [comment]
No, deleting TSM data on Google Drive doesn't do anything with this bug.
Maybe it worked a year ago, but with the current app version - no.

![2021-04-26_112443](https://user-images.githubusercontent.com/62963044/116052634-7becae00-a682-11eb-8d78-7cc6dea89916.jpg)

I click on the button with the cloud and yellow exclamation mark - and nothing happens...

P.S. Oh, just mentioned you message about the fix in the next update.
Will wait for it. Thank you!

**Comment by andynaz** [comment]
I had an error while syncing: I was stuck in a 403 Forbidden error in connecting to Google Drive.
Deleting remote data, closing Firefox and re-trying solved the issue.

Thanks
